docs-2.0/3.ngql-guide/17.query-tuning-statements/1.explain-and-profile.md

```diff
@@ -1,97 +1,99 @@
+    |  0 | Start    |              |                | outputVar: [{"colNames":[],"name":"__Start_0","type":"DATASET"}]    |
+    -----+----------+--------------+----------------+----------------------------------------------------------------------
+    ```
+
+* `PROFILE`
+
+    ```ngql
+    nebula> PROFILE format="row" SHOW TAGS;
+    +--------+
+    | Name   |
+    +--------+
+    | player |
+    +--------+
+    | team   |
+    +--------+
+    Got 2 rows (time spent 2038/2728 us)
+
+    Execution Plan
+
+    -----+----------+--------------+----------------------------------------------------+----------------------------------------------------------------------
+    | id | name     | dependencies | profiling data                                     | operator info                                                       |
+    -----+----------+--------------+----------------------------------------------------+----------------------------------------------------------------------
+    |  1 | ShowTags | 0            | ver: 0, rows: 1, execTime: 42us, totalTime: 1177us | outputVar: [{"colNames":[],"name":"__ShowTags_1","type":"DATASET"}] |
+    |    |          |              |                                                    | inputVar:                                                           |
+    -----+----------+--------------+----------------------------------------------------+----------------------------------------------------------------------
+    |  0 | Start    |              | ver: 0, rows: 0, execTime: 1us, totalTime: 57us    | outputVar: [{"colNames":[],"name":"__Start_0","type":"DATASET"}]    |
+    -----+----------+--------------+----------------------------------------------------+----------------------------------------------------------------------
+    ```
+
+The descriptions are as follows.
+
+|Parameter|Description|
 |-|-|
-|`id`|Indicates the ID of the operator.|
-|`name`|Indicates the name of the operator.|
-|`dependencies`|Shows the ID of the operator that the current operator depends on.|
-|`profiling data`|Shows the execution profile. `ver` is the version of the operator, which you can use to identify loops; `rows` shows the number of rows to be output by the operator; `execTime` shows the execution time only; `totalTime` contains the execution time and the system scheduling and queueing time.|
-|`operator info`|Shows the detailed information of the operator.|
+|`id`|The ID of the `operator`.|
+|`name`|The name of the `operator`.|
+|`dependencies`|The ID of the `operator` that the current `operator` depends on.|
+|`profiling data`|The content of the execution profile. `ver` is the version of the `operator`. `rows` shows the number of rows to be output by the `operator`. `execTime` shows the execution time of `action`. `totalTime` is the sum of the execution time, the system scheduling time, and the queueing time.|
```

Comment:
`action`;

Reply:
Semicolons are seldom used in our docs, so I guess the full stop here is ok.